modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/endpoint/WorkflowRestService.java

```diff
@@ -455,8 +455,8 @@ public Response getWorkflowsAsXml(@QueryParam("state") List<String> states, @Que
           @RestParameter(name = "seriesTitle", isRequired = false, description = "Filter results by series title", type = STRING),
           @RestParameter(name = "creator", isRequired = false, description = "Filter results by the mediapackage's creator", type = STRING),
           @RestParameter(name = "contributor", isRequired = false, description = "Filter results by the mediapackage's contributor", type = STRING),
-          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by workflow start date.", type = STRING),
-          @RestParameter(name = "todate", isRequired = false, description = "Filter results by workflow start date.", type = STRING),
+          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by the creation date metadata field of the media package which is usually populated via the `temporal` Dublin Core element. Use UTC dates in the format <pre>2020-12-24T08:30:00Z</pre>.", type = STRING),
+          @RestParameter(name = "todate", isRequired = false, description = "Filter results by the creation date metadata field of the media package which is usually populated via the `temporal` Dublin Core element. Use UTC dates in the format <pre>2020-12-24T08:30:00Z</pre>.", type = STRING),
```

Comment:
Please don't hate me. \*ducks\*

```suggestion
          @RestParameter(name = "todate", isRequired = false, description = "Filter results by the bibliographic start date of the event, which maps to the `temporal` field in the Dublin Core catalog. Use UTC dates in the format <pre>2020-12-24T08:30:00Z</pre>.", type = STRING),
```

Reply:
The code actually looks at the date attribute of the media package and not at any metadata directly. That field is populated by values from the Dublin Core catalog (either `temporal` or (if `temporal` does not exist) `created`). That's why I don't agree with your phrasing. But since we can't find a common ground here, let's leave it as it was.